apps/api/src/app/widgets/dtos/feeds-response.dto.ts

```diff
@@ -0,0 +1,131 @@
+import { ApiExtraModels, ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { ActorTypeEnum, ChannelTypeEnum, IActor, INotificationDto } from '@novu/shared';
+
+import { SubscriberResponseDto } from '../../subscribers/dtos';
+import { EmailBlock, MessageCTA } from './message-response.dto';
+
+class Actor implements IActor {
+  @ApiProperty()
+  data: string | null;
+
+  @ApiProperty({ enum: ActorTypeEnum })
+  type: ActorTypeEnum;
+}
+
+@ApiExtraModels(EmailBlock, MessageCTA)
+export class NotificationDto implements INotificationDto {
+  @ApiPropertyOptional()
+  _id: string;
+
+  @ApiProperty()
+  _templateId: string;
+
+  @ApiProperty()
+  _environmentId: string;
+
+  @ApiProperty()
+  _messageTemplateId: string;
+
+  @ApiProperty()
+  _organizationId: string;
+
+  @ApiProperty()
+  _notificationId: string;
+
+  @ApiProperty()
+  _subscriberId: string;
+
+  @ApiProperty()
+  _feedId: string;
+
+  @ApiProperty()
+  _jobId: string;
+
+  @ApiPropertyOptional()
+  createdAt: string;
+
+  @ApiPropertyOptional()
+  updatedAt: string;
+
+  @ApiPropertyOptional()
+  expireAt: string;
+
+  @ApiPropertyOptional({
+    type: Actor,
+  })
+  actor?: Actor;
+
+  @ApiPropertyOptional({
+    type: SubscriberResponseDto,
+  })
+  subscriber?: SubscriberResponseDto;
```

Comment:
What’s the use case for returning the subscriber DTO in the feed?

Reply:
This dto is just what we had before, I've only fixed some types, for example for content.
The improvements to the endpoint itself we will do separately.